torch/csrc/jit/codegen/cuda/scheduler/normalization.cpp

```diff
@@ -715,6 +719,10 @@ TORCH_CUDA_CU_API c10::optional<ReductionParams> getPersistentHeuristics(
       persistent_buffer_size_info.persistent_buffer_size,
       persistent_buffer_size_info.projected_persistent_buffer_size);
 
+  bool project_persistent_buffers =
+      persistent_buffer_size_info.projected_persistent_buffer_size <
+      persistent_buffer_size_info.persistent_buffer_size;
+
```

Comment:
Having some comments with a concrete example would be helpful to understand the context of this logic (e.g., caching a half-type input tensor rather than its tensor converted to float).